pkg/server/plugin/nodeattestor/awsiid/iid.go

```diff
@@ -106,14 +110,14 @@ type IIDAttestorConfig struct {
 	AssumeRole                      string   `hcl:"assume_role"`
 	pathTemplate                    *agentpathtemplate.Template
 	trustDomain                     spiffeid.TrustDomain
-	awsCAPublicKey                  *rsa.PublicKey
+	getAWSCACertificate             func(string, PublicKeyType) (*x509.Certificate, error)
```

Comment:
We talked about maybe providing a way in the config for operators to specify an override for the signing certificate for a given region. Is that not needed anymore due to the very long expiration dates on the RSA2048 CAs?

Reply:
We talked about providing a way to specify a cert for region, and my understanding is that we prefer to have all the certs and select automatically which one to use based on the region, which I think is not bad idea considering the very long expiration dates.